zebra-consensus/src/transaction/check.rs

```diff
@@ -208,3 +209,45 @@ where
 
     Ok(())
 }
+
+/// Pre-Heartwood: returns true.
+/// Heartwood-onward: returns if all Sapling or Orchard outputs, if any, decrypt successfully with
+/// an all-zeroes outgoing viewing key.
```

Comment:
```suggestion
/// Pre-Heartwood: returns `Ok`.
/// Heartwood-onward: returns `Ok` if all Sapling or Orchard outputs, if any, decrypt successfully with
/// an all-zeroes outgoing viewing key. Returns `Err` otherwise.
```